### PR TITLE
Minor changes to integtests

### PR DESCRIPTION
**Pull request by eflumerf**
Copy of #148 targeting a prep-release branch

### CODE REVIEW

**Comment by eflumerf** [comment]
minimal_system_quick_test.py passes, new tests show up in daqsystemtest_integtest_bundle.sh. (Since I opened this PR as a copy of #148, I can't approve it)

**Review by bieryAtFnal — APPROVED** [review]
LGTM, after addition of no:cacheprovider to the DBT_AREA_ROOT case, when that directory is not writable.

**Pull request merged**
